local-volume/provisioner/pkg/util/volume_util.go

```diff
@@ -94,6 +98,16 @@ func (u *volumeUtil) DeleteContents(fullPath string) error {
 	return nil
 }
 
+// GetFsAvailable returns available capacity about a mounted filesystem. fullPath
+// is the pathname of any file within the mounted filesystem.
+func (u *volumeUtil) GetFsAvailable(fullPath string) (uint64, error) {
+	var s syscall.Statfs_t
+	if err := syscall.Statfs(fullPath, &s); err != nil {
+		return 0, err
+	}
+	return uint64(s.Frsize) * s.Bavail, nil
```

Comment:
There is no such a field that i know of.  Looking further into df, I think what we can do is to use 'used + available'.

related logic from df:
```c
  bv->total = fsu->fsu_blocks;
  bv->available_to_root = fsu->fsu_bfree;

  bv->used = bv->total - fsu->fsu_bfree;

  uintmax_t nonroot_total = v->used + v->available;
```

so simply put, we can return `s.Blocks - s.Bfree + s.Bavail`.  This will return total capacity for non root user, removed system reserved altogether.

WDYT